src/components/Header/index.js

```diff
@@ -4,29 +4,34 @@ import { createUseStyles } from 'react-jss'
 const useStyles = createUseStyles({
```

Comment:
Can we separate the styling into a separate file and import it here?
It will be easier to manage the styling and functional component.

Reply:
Sure thing! I have now moved it to another file. I'd like to make it responsive but I'm waiting for Chali to provide a figma design for me to follow.